datastore/settings/settings_dev.py

```diff
@@ -1,14 +1,41 @@
 # flake8: noqa
+import os
 from settings.settings import *
 
 # This adds the CORS header to API calls for the django dev server
 
 DEBUG = True
 
+SECRET_KEY = "itsasecret"
```

Comment:
could we use something slightly less static like `__path__` or HOSTNAME + something or something, just for the case where we've got multiple dev instances running. Also certain bots/people probing this repo will look for strings like this and may decide to try (and fail to) and exploit it in live.

Reply:
In the end I've used `hostname` + `os.getcwd()` because __path__ seems to be undefined in the context of runserver :shrug: